openedx/core/djangoapps/programs/tests/test_signals.py

```diff
@@ -224,3 +224,60 @@ def test_programs_enabled(self, mock_is_learner_issuance_enabled, mock_task):
         self.assertEqual(mock_is_learner_issuance_enabled.call_count, 1)
         self.assertEqual(mock_task.call_count, 1)
         self.assertEqual(mock_task.call_args[0], (TEST_USERNAME, TEST_COURSE_KEY))
+
+
+# The credentials app isn't installed for the CMS.
+@skip_unless_lms
+@mock.patch('openedx.core.djangoapps.programs.tasks.v1.tasks.update_certificate_visible_date_on_course_update.delay')
+@mock.patch(
+    'openedx.core.djangoapps.credentials.models.CredentialsApiConfig.is_learner_issuance_enabled',
+    new_callable=mock.PropertyMock,
+    return_value=False,
+)
+class CourseCertAvailableDateChangedReceiverTest(TestCase):
+    """
+    Tests for the `handle_course_cert_date_change` signal handler function.
+    """
+
+    @property
+    def signal_kwargs(self):
+        """
+        DRY helper.
+        """
+        return dict(
+            sender=self.__class__,
+            course_key=TEST_COURSE_KEY,
+        )
```

Comment:
Make it a class-level variable as it will remain same. No need to create the property. Plus, use {} for creating dictionary as `dict` method is expensive

Reply:
We need the `self.__class__` value for `sender`. Should I create an instance variable instead?

Reply:
Just change dict to explicit {}. See [this](https://doughellmann.com/blog/2012/11/12/the-performance-impact-of-using-dict-instead-of-in-cpython-2-7-2/).